src/plugins.js

```diff
@@ -41,7 +44,7 @@ export function getCsrfPlugin(secret) {
 	const register = (server, options, next) => {
 		const cookieOptions = {
 			path: '/',
-			isSecure: process.env.NODE_ENV === 'production',
+			isSecure: config.get('isProd'),
```

Comment:
related - use `server.app.csrf_secret` for `options.secret` on line 67 instead of passing it in as an argument. That way, you don't have to pass `config` into `getPlugins` at all